### PR TITLE
Update Build Guides with Qt 5.6.2

### PR DESCRIPTION
**Pull request by anshuman64**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: c82bd67a1491fbf2adcbc25d1596c8c603bb43dc available at: https://gist.github.com/ad8899328ad87b667d9a114b3880265b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-c82bd67.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-c82bd67.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ed6211fcb7bd4b0dc954abae13959593a146a89d available at: https://gist.github.com/ad8899328ad87b667d9a114b3880265b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-ed6211f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-ed6211f.dmg

File: BUILD_WIN.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ Download and install the CMake 3.8.0-rc2 "win64-x64 Installer" from the [CMake W
...
 
-Make sure to select all components when going through the installer.
+Keep the default components checked when going through the installer.
 
 ###Step 4. Setting Qt Environment Variable
 
 Go to "Control Panel > System > Advanced System Settings > Environment Variables > New..." (or search “Environment Variables” in Start Search).
 * Set "Variable name": QT_CMAKE_PREFIX_PATH
-* Set "Variable value": `C:\Qt\Qt5.6.1\5.6\msvc2013_64\lib\cmake`
+* Set "Variable value": `C:\Qt\Qt5.6.2\5.6\msvc2013_64\lib\cmake`
```
Comment (zfox23) [inline BUILD_WIN.md]: Not sure how I didn't notice this on the old build guide - but this should be replaced with something like:
`Set "Variable value": <QT 5.6.2 Install Path>\5.6\msvc2013_64\lib\cmake`

Comment (anshuman64) [inline BUILD_WIN.md]: Going with the "don't fix what isn't broken" and "tailor to the least tech-savvy user and let the smart ones figure it out" methodologies, I think what's there is fine.

Comment (zfox23) [inline BUILD_WIN.md]: But then, down below, you use the `%HIFI_DIR%` syntax. Don't you think the same user would be able to figure out `%QT_DIR%`?

More importantly - I'm pretty sure this isn't the default QT install directory, so these instructions require people to think on their feet anyway.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 54e4eb5eae364c994a9c92c8b441c90738484a6e available at: https://gist.github.com/ad8899328ad87b667d9a114b3880265b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-54e4eb5.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-54e4eb5.dmg

File: BUILD_WIN.md (hunk 1 above)
Comment (anshuman64) [inline BUILD_WIN.md]: You're totally right regarding the fact that we're already using %HIFI_DIR% syntax. Made the change.

**Comment by zfox23** [comment]
Woohoo! Dev-approved.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 521db82ed0ddc237e9e6a6608b31a6c5877175be available at: https://gist.github.com/ad8899328ad87b667d9a114b3880265b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-521db82.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10287-521db82.dmg

**Pull request merged**